rakefile.rb

```diff
@@ -1,6 +1,8 @@
+CYM_BSP_VERSION = "0.2.2-dev"
 PROJECT_ROOT = File.dirname(__FILE__)
```

Comment:
The BSP version has been incremented to 0.2.3, so this should probably be 0.2.3-dev